src/fileFactories/DummyFileFactory.js

```diff
@@ -0,0 +1,12 @@
+import BaseFileFactory from './BaseFileFactory'
+
+export default class DummyFileFactory extends BaseFileFactory {
```

Comment:
Actually, maybe move this declaration to `DummyFileFactory.test.js` because we still need to assert "if we extend filefactory, it should have x y z attribute"?

Reply:
Yes, Ill clean this up and then we can refine it later and add more of those tests.